src/ListConfig/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { ShadedColor, colors } from "../colors";
+
+/**
+ * Enumeration of all icon sizes
+ *
+ * This uses string literals instead of a TypeScript union so we can use
+ * strings, like `"small"`, for props instead of `IconSize.small`
+ */
+type IconSize = "small" | "normal" | "large";
+
+interface ListConfig {
+  /**
+   * Override the the default element used to render `endIcon`
+   *
+   * All props provided will be merged with props that this component adds,
+   * including `className`s being merged.
+   *
+   * @default <div />
+   */
+  endIconAs: React.ReactElement;
+
+  /**
+   * Background color of the hovered item in a menu
+   *
+   * The text color will be determined automatically
+   */
+  hoverColor: ShadedColor;
```

Comment:
Is this in response to https://meteor.slack.com/archives/CJM405HJB/p1592282599095400 ?  
I wonder if this might need to vary by individual items?  
E.g. some items might have a blue hover color, and some might be red, depending on what the individual item is?
pinging @Jephuff iirc you PR'ed the fix for this in the app

Reply:
It's not; it's just with the goal of being able to make this work in dark or light mode with a single configuration option.